src/test/regress/sql/multi_name_lengths.sql

```diff
@@ -215,8 +207,19 @@ SELECT master_create_distributed_table('sneaky_name_lengths', 'int_col_123456789
 SELECT master_create_worker_shards('sneaky_name_lengths', '2', '2');
 
 \c - - :public_worker_1_host :worker_1_port
-\di public.sneaky*225030
```

Comment:
The hardcoded shardId here makes it problematic to update some of the tests above. That's why I am doing a lookup to find an index name on a shard.

I could have also write a query that is similar to `/di <index_name>`. However I think this is ok for now.